src/main/java/uk/gov/companieshouse/confirmationstatementapi/model/response/CompanyValidationResponse.java

```diff
@@ -1,25 +1,25 @@
 package uk.gov.companieshouse.confirmationstatementapi.model.response;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
-import uk.gov.companieshouse.confirmationstatementapi.eligibility.EligibilityFailureReason;
+import uk.gov.companieshouse.confirmationstatementapi.eligibility.EligibilityStatusCode;
 
-public class EligibilityFailureResponse {
+public class CompanyValidationResponse {
 
     @JsonProperty("validation_error")
```

Comment:
you're going to want to rename this field to match the spec: https://github.com/companieshouse/confirmation-statement-api/blob/main/spec/schema.json#L167

This is as its the same object for success or failure.
It's not just validation errors anymore